src/peakrdl_regblock/udps/fixedpoint.py

```diff
@@ -0,0 +1,100 @@
+from typing import Any
+
+from systemrdl.component import Field
+from systemrdl.node import Node, FieldNode
+from systemrdl.udp import UDPDefinition
+
+
+class FixedpointWidth(UDPDefinition):
```

Comment:
nitpick, make this a "private" class
```suggestion
class _FixedpointWidth(UDPDefinition):
```